src/control/server/instance_exec.go

```diff
@@ -147,15 +147,17 @@ func (srv *IOServerInstance) run(ctx context.Context, membership *system.Members
 		return
 	}
 
-	if err = srv.start(ctx, errChan); err != nil {
-		return
-	}
-	if srv.isMSReplica() {
-		// MS bootstrap will not join so register manually
-		if err := srv.registerMember(membership); err != nil {
+	// After we know that the instance storage is ready, fire off
```

Comment:
think this block could be moved to the end of IOServerInstance.format()

Reply:
Done.